src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java

```diff
@@ -0,0 +1,208 @@
+
+    @MediaType(value = ANY, strict = false)
+    @DisplayName("Get Tier Config")
+    public CBCTierConfig getTierConfig(
+        @Connection CBCConnection connection,
+        @ParameterGroup(name="Tier Config ID") ResourceActionParameter getTierConfigParameter
+    ) throws CBCException {
+        return (CBCTierConfig) connection
+            .newQ(new TypeReference<CBCTierConfig>() {})
+            .collection("tier").collection("configs", getTierConfigParameter.getId())
+            .get();
+        }
+
+    @MediaType(value = ANY, strict = false)
+    @DisplayName("Get Tier Config Request")
+    public CBCTierConfigRequest getTierConfigRequest(
+        @Connection CBCConnection connection,
+        @ParameterGroup(name="Tier Config Request ID") ResourceActionParameter getTierConfigParameter
+    ) throws CBCException {
+        return (CBCTierConfigRequest) connection
+            .newQ(new TypeReference<CBCTierConfigRequest>() {})
+            .collection("tier").collection("config-requests", getTierConfigParameter.getId())
+            .get();
+        }
+
+    @MediaType(value = ANY, strict = false)
+    @DisplayName("Create Tier Config Request")
+    public CBCTierConfigRequest  createTierConfigRequest(
+        @Connection CBCConnection connection,
+        @ParameterGroup(name="Create Tier Config Request")
+        NewTierConfigParameter newRequestParameter
+    ) throws CBCException {
+        return (CBCTierConfigRequest) connection.newQ(new TypeReference<CBCTierConfigRequest>() {})
+            .collection("tier").collection("config-requests")
+            .create(newRequestParameter.buildEntity());
+        }    
+
+    @MediaType(value = ANY, strict = false)
+    @DisplayName("Approve Tier Config Request")
+    public CBCTierConfigRequest approveTierConfigRequest(
+        @Connection CBCConnection connection,
+        @ParameterGroup(name="Approve Tier Config Request")
+        ApproveTierConfigRequestParameter newRequestParameter
+    ) throws CBCException {
+        return (CBCTierConfigRequest) connection.newQ(new TypeReference<CBCTierConfigRequest>() {})
+            .collection("tier")
+            .collection("config-requests",newRequestParameter.getTierConfigRequestId())
+            .collection("approve")
+            .create(newRequestParameter.buildEntity());
+        }    
+        
+    @MediaType(value = ANY, strict = false)
+    @DisplayName("Inquire Tier Config Request")
+    public CBCTierConfigRequest inquireTierConfigRequest(
+        @Connection CBCConnection connection,
+        @ParameterGroup(name="Inquire Tier Account Request")
+        ActionTierConfigRequestParameter newRequestParameter
+    ) throws CBCException {
+        return (CBCTierConfigRequest) connection.newQ(new TypeReference<CBCTierConfigRequest>() {})
+            .collection("tier")
+            .collection("config-requests",newRequestParameter.getTierConfigRequestId())
+            .action("inquire", HttpMethod.POST,"");
```

Comment:
.action("inquire", HttpMethod.POST, null);

And similar in below code as well.

Reply:
Done